drivers/flash/spi_nor.c

```diff
@@ -540,6 +540,45 @@ static int spi_nor_wrsr(const struct device *dev,
 	return ret;
 }
 
+#define MX25R_HIGH_PERF_MODE BIT(1)
+
+#ifdef CONFIG_SPI_NOR_HIGH_PERF_MODE
+static int enable_high_perf_mode(const struct device *dev, uint8_t jedec_id[])
+{
+	int sr, ret;
+	uint8_t data[3];
+
+	/* Only supported on Macronix MX25R Ultra Low Power series. */
```

Comment:
Is there any way to inform the user about the MX25R-only support without them having to go into the driver code?